frontend/components/JoinGravePanel.tsx

```diff
@@ -194,7 +197,7 @@ const JoinGravePanel: React.FC = () => {
                 {step.completeText.text}
                 {step.completeText.address && (
                   <a
-                    href={`${constants.LUKSO_EXPLORER.TESTNET.ADDRESS}${step.completeText.address}`}
+                    href={`${networkConfig.luksoExplorer}${step.completeText.address}`}
```

Comment:
@democrazyfi there's issue here. `networkConfig.luksoExplorer` does not have `/address/` path that this previously relied on. i think this is the case in a few other cases.